cmd/operator_uninstall.go

```diff
@@ -38,7 +38,7 @@ func newOperatorUninstallCmd(operatorConfig *operatorCommandConfig) *cobra.Comma
 		Short: "Uninstall the Appsody Operator.",
 		Long:  `Uninstall the Appsody Operator from your configured Kubernetes cluster.`,
 		Example: `  appsody operator uninstall --namespace my-namespace
-  Uninstalls the Appsody Operator in the 'my-namespace- namespace from your Kubernetes cluster.`,
+  Uninstalls the Appsody Operator in the "my-namespace-"namespace from your Kubernetes cluster.`,
```

Comment:
Why is this written as:
... "my-namespace-"namespace...
and not
... "my-namespace" namespace...
?

Reply:
I have fixed this in the most recent commit